redbot/core/core_commands.py

```diff
@@ -4100,15 +4084,14 @@ async def command_default_enable_cog(self, ctx: commands.Context, *, cogname: st
         **Arguments:**
             - `<cogname>` - The name of the cog to make enabled by default. Must be title-case.
         """
-        cog = self.bot.get_cog(cogname)
-        if not cog:
-            return await ctx.send(_("Cog with the given name doesn't exist."))
+        cog = cogname
+        cogname = cog.qualified_name
         await self.bot._disabled_cog_cache.default_enable(cogname)
         await ctx.send(_("{cogname} has been set as enabled by default.").format(cogname=cogname))
 
     @commands.guild_only()
     @command_manager.command(name="disablecog")
-    async def command_disable_cog(self, ctx: commands.Context, *, cogname: str):
+    async def command_disable_cog(self, ctx: commands.Context, *, cogname: CogConverter):
```

Comment:
As with command_name->command, this should also be renamed from cogname to cog:
```suggestion
    async def command_disable_cog(self, ctx: commands.Context, *, cog: CogConverter):
```
You will need to update the docstring and docs as well.